lib/ansible/modules/cloud/amazon/elasticache_parameter_group.py

```diff
@@ -296,8 +313,8 @@ def main():
     exists = get_info(connection, parameter_group_name)
 
     # check that the needed requirements are available
-    if state == 'present' and not (exists and parameter_group_family and group_description):
-        module.fail_json(msg="Creating a group requires a family group and a description.")
+    if state == 'present' and not exists and not parameter_group_family:
```

Comment:
This can also be written as `if state == 'present' and not (exists or parameter_group_family):`